protos/cerbos/cloud/store/v1/store.proto

```diff
@@ -74,6 +74,15 @@ message GetFilesResponse {
   repeated File files = 2;
 }
 
+message GetCurrentVersionRequest {
+  string store_id = 1 [(buf.validate.field).string.len = 12];
+}
+
+message GetCurrentVersionResponse {
+  int64 store_version = 1;
+  optional ChangeDetails change_details = 2;
```

Comment:
I don't think this would be `optional` because we always add change details?

Reply:
🤔  `change_details` parameters are optional in requests:

[ModifyFilesRequest](https://github.com/cerbos/cloud-api/blob/abf278ad268c6f950efcad821fa6e47d128609a6/protos/cerbos/cloud/store/v1/store.proto#L135)

[ReplaceFilesRequest](https://github.com/cerbos/cloud-api/blob/abf278ad268c6f950efcad821fa6e47d128609a6/protos/cerbos/cloud/store/v1/store.proto#L183)

Reply:
Yes, but this is a response. IIRC we always add change details but, I guess it _could_ be unset and having a presence check might be useful.